plasmapy/atomic/particles.py

```diff
@@ -0,0 +1,219 @@
+
+    for thing in everything:
+        if 'half-life' not in Particles[thing].keys():
+            Particles[thing]['half-life'] = np.inf * u.s
+
+    for particle in particles:
+        Particles[particle]['antimatter'] = False
+
+    for antiparticle in antiparticles:
+        Particles[antiparticle]['antimatter'] = True
+
+    return Particles
+
+
+def _create_alias_dicts(Particles):
+    """Create dictionaries for case sensitive aliases and case
+    insensitive aliases."""
+
+    case_sensitive_aliases = {}
+    case_insensitive_aliases = {}
+
+    for symbol in Particles.keys():
+        name = Particles[symbol]['name']
+        case_insensitive_aliases[name.lower()] = symbol
+
+    case_sensitive_aliases_for_a_symbol = [
+        (['beta-'], 'e-'),
+        (['beta+'], 'e+'),
+        (['p+'], 'p'),
+        (['n-1'], 'n'),
+        (['H-2'], 'D'),
+        (['H-2+', 'H-2 1+', 'H-2 +1'], 'D 1+'),
+        (['H-3+', 'H-3 1+', 'H-3 +1'], 'T 1+'),
+    ]
+
+    case_insensitive_aliases_for_a_symbol = [
+        (['antielectron'], 'e+'),
+        (['muon-'], 'mu-'),
+        (['muon+'], 'mu+'),
+        (['tau particle'], 'tau-'),
+        (['protium'], 'H-1'),
+        (['protium+', 'protium 1+', 'protium +1'], 'p'),
+        (['deuterium', 'hydrogen-2'], 'D'),
+        (['deuteron', 'deuterium+', 'deuterium 1+', 'deuterium +1', 'D+'],
+         'D 1+'),
+        (['tritium', 'hydrogen-3'], 'T'),
+        (['triton', 'tritium+', 'tritium 1+', 'tritium +1'], 'T 1+'),
+        (['alpha'], 'He-4 2+'),
+    ]
+
+    for aliases, symbol in case_sensitive_aliases_for_a_symbol:
+        for alias in aliases:
+            case_sensitive_aliases[alias] = symbol
+
+    for aliases, symbol in case_insensitive_aliases_for_a_symbol:
+        for alias in aliases:
+            case_insensitive_aliases[alias.lower()] = symbol
+
+    alias_keys = list(case_insensitive_aliases.keys())
+
+    for alias in alias_keys:
+        if 'anti' in alias and 'anti-' not in alias:
+            symbol = case_insensitive_aliases[alias].lower()
+            new_alias = alias.replace('anti', 'anti-')
+            case_insensitive_aliases[new_alias] = symbol
+
+    return case_sensitive_aliases, case_insensitive_aliases
+
+
+_Particles = _create_Particles_dict()
+
+_case_sensitive_aliases, _case_insensitive_aliases = \
+    _create_alias_dicts(_Particles)
+
+
+def _get_standard_symbol(alias):
+    """If the input is a recognized alias for a particle, then
+    this function will return the standard symbol associated
+    with that alias.  Otherwise, this function will return the
+    original string."""
+
+    if not isinstance(alias, str):
+        return alias
+
+    if alias in _case_sensitive_aliases.keys():
+        return _case_sensitive_aliases[alias]
+    elif alias.lower() in _case_insensitive_aliases.keys():
+        return _case_insensitive_aliases[alias.lower()]
+    else:
+        return alias
+
+
+print(_case_insensitive_aliases)
```

Comment:
Put this in to an `if __name__=="__main__":` block, or it's going to print the aliases on every import of this module!